src/internal/Subscriber.ts

```diff
@@ -235,4 +189,12 @@ export class SafeSubscriber<T> extends Subscriber<T> {
       super.unsubscribe();
     }
   }
-}
\ No newline at end of file
+}
+
+function handleError(err: any) {
+  if (config.useDeprecatedSynchronousErrorHandling) {
+    throw err;
+  } else {
+    reportUnhandledError(err);
+  }
```

Comment:
```suggestion
  if (config.useDeprecatedSynchronousErrorHandling) {
    throw err;
  }

  reportUnhandledError(err);
```

Reply:
I saw this too late and had already updated the code. Good find! I've added it manually.